client/package.json

```diff
@@ -2,8 +2,11 @@
     "dependencies": {
         "react": "15.6.1",
         "react-dom": "15.6.1",
+        "react-router": "^4.1.1",
         "react-router-dom": "^4.1.1",
-        "material-ui": "^0.20.0"
+        "redux": "3.7.2",
```

Comment:
We probably want `^3.7.2`, so that npm can load the newest version that's compatible.